api/server/httputils/form.go

```diff
@@ -162,3 +162,22 @@ func DecodePlatform(platformJSON string) (*ocispec.Platform, error) {
 
 	return &p, nil
 }
+
+// DecodePlatforms decodes the OCI platform JSON string into a Platform struct.
+//
+// Typically, the argument is a value of: r.Form["platform"]
+func DecodePlatforms(platformJSONs []string) ([]ocispec.Platform, error) {
```

Comment:
Heh, I thought we already had this, but it really now that for save/load we didn't yet implement multiple platforms, so we currently still produce an error; https://github.com/moby/moby/blob/7937f0846c130e91836f5f640c41ce12d07da4c8/api/server/router/image/image_routes.go#L279-L291